### DISCUSSION TITLE
https://ais-web.test.dp.tech/datasets/detail?pageType=datasets&name=H2O-Phase-Diagram&id=1

### DISCUSSION THREAD

**Discussion by giscus** [yanyumei/giscus-test#14] [Announcements]
# https://ais-web.test.dp.tech/datasets/detail?pageType=datasets&name=H2O-Phase-Diagram&id=1

https://ais-web.test.dp.tech/datasets/detail?pageType=datasets&name=H2O-Phase-Diagram&id=1

**Comment by Mile-Away** [comment]
如果我没有连接 GitHub，我可以发布出来吗？

**Reply by Mile-Away** [reply]
这是发出来了吗？

**Reply by smallplum** [reply]
发出来了

**Comment by smallplum** [comment]
添加一条评论
